### PR TITLE
Adding egret to install_requires

### LINKED ISSUE #21: Add Egret as part of setup

### PR DESCRIPTION
**Pull request by bknueven**
Closes #21.

### CODE REVIEW

**Comment by bknueven** [comment]
@jsiirola could you take a look at this?

File: setup.py [hunk 1]
```diff
@@ -41,5 +41,7 @@
         },
       package_data={'prescient.downloaders.rts_gmlc_prescient':['runners/*.txt','runners/templates/*']},
       install_requires=['numpy','matplotlib','pandas','scipy','pyomo','six',
-                        'pyutilib', 'python-dateutil', 'networkx']
+                        'pyutilib', 'python-dateutil', 'networkx',
+                        'egret @ git+https://github.com/grid-parity-exchange/Egret.git'],
+      dependency_links=['git+https://github.com/grid-parity-exchange/Egret.git#egg=egret'],
```
Comment (jsiirola) [inline setup.py:46]: This might be more stable if you pick a package name that does not exist on PyPI.  Maybe `egret-grid`?

Comment (bknueven) [reply setup.py:46]: That seems to cause other issues. On my system, adding `dependency_links` lets `setuptools.setup` find the GitHub Egret.

Changing the name causes other issues on my system:
```
Collecting egret-grid@ git+https://github.com/grid-parity-exchange/Egret.git
  Cloning https://github.com/grid-parity-exchange/Egret.git to /private/var/folders.  /3t/hd_9tsvj1qvd9hlqn7f4yh1hg_1n97/T/pip-install-2lt6j196/egret-grid
  WARNING: Generating metadata for package egret-grid produced metadata for project name egret. Fix your #egg=egret-grid fragments.
ERROR: Exception:
Traceback (most recent call last):
...
    assert req_to_install.user_supplied
AssertionError
```

Comment (jsiirola) [reply setup.py:46]: Ick.  OK - so we need to keep the project name in sync with the Egret source.  I suspect that what you have here is the best we can do until Egret has a formal PyPI release.

**Pull request merged**